website/content/en/docs/upgrading/upgrade-guide.md

```diff
@@ -88,7 +88,7 @@ Below is the full changelog for v1, copied from the [v1 Migration Upgrade Proced
 * Karpenter now adds a readiness status condition to the EC2NodeClass. Make sure to upgrade your Custom Resource Definitions before proceeding with the upgrade. Failure to do so will result in Karpenter being unable to provision new nodes.
 * Karpenter no longer updates the logger name when creating controller loggers. We now adhere to the controller-runtime standard, where the logger name will be set as `"logger": "controller"` always and the controller name will be stored in the structured value `"controller"`
 * Karpenter updated the NodeClass controller naming in the following way: `nodeclass` -> `nodeclass.status`, `nodeclass.hash`, `nodeclass.termination`
-* Karpenter's NodeClaim status conditions no longer include the `severity` field
+* Starting with `0.37.3` Karpenter's NodeClaim status conditions no longer include the `severity` field
```

Comment:
Is this true? Isn't this starting from the first minor version?

Reply:
I'm assuming you might want to have the same statement around the webhook being enabled here

Reply:
Typo - fixing.